identity-apps-core/apps/x509-certificate-authentication-portal/src/main/webapp/includes/cookie-consent-banner.jsp

```diff
@@ -0,0 +1,57 @@
+<%--
+  ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+  ~
+  ~ WSO2 LLC. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+--%>
+
+<%@ page import="org.apache.commons.text.StringEscapeUtils" %>
+<%@ page import="org.apache.commons.lang.StringUtils" %>
+<%@ page import="org.wso2.carbon.identity.mgt.endpoint.util.IdentityManagementEndpointUtil" %>
+
+<%-- Include tenant context --%>
+<jsp:directive.include file="../tenant-resolve.jsp"/>
+
+<%-- Branding Preferences --%>
+<jsp:directive.include file="./branding-preferences.jsp"/>
+
+<%-- Localization --%>
+<jsp:directive.include file="./localize.jsp" />
+
+<%
+    if (!StringUtils.isBlank(cookiePolicyURL)) {
+%>
+<div id="cookie-consent-banner" data-testid="cookie-consent-banner" class="ui segment cookie-consent-banner inverted aligned-right hidden transition">
+    <div class="banner-image" data-testid="cookie-consent-banner-image">
+        <div class="theme-icon floated-left hover-rounded tiny transparent cookie-icon" data-testid="generic-icon">
+            <svg class="icon" width="14" height="14" viewBox="0 0 514.374 514.374">
```

Comment:
Will it be possible to get this icon from theme instead of hardcoding here?

Reply:
I see this as a workaround to minimize loading delays. Since the cookie banner is displayed instantly when the webpage loads, any delay in loading the banner image could cause a glitch, which might be noticeable to users. Therefore, embedding it directly into the code helps mitigate this issue. We're employing a similar approach on our loading pages as well. https://github.com/wso2/identity-apps/blob/eda909b8fd71ac6695d4b28631816bc0c79c0189/identity-apps-core/apps/authentication-portal/src/main/webapp/oauth_response.jsp#L505-L526